src/_scss/pages/dashboard/_dashboardTabs.scss

```diff
@@ -0,0 +1,19 @@
+.dashboard-tabs {
+    background-color: black;
+    .dashboard-tabs__content {
+        @include display(flex);
+        @include justify-content(center);
+        .dashboard-tabs__button {
```

Comment:
Just for my knowledge, what do the double underscores in these files indicate? I realize it's just a naming convention, I'm just curious

Reply:
Yep, that's part of the [BEM](http://getbem.com/introduction/) (Block Element Modifier) naming convention. Here, the "block" is `dashboard-tabs` and the button is an "element" of that block.